resources/views/dashboard/getting-started.blade.php

```diff
@@ -1,56 +1,78 @@
+                </div>
+                <x-interpretation name="{{ __('Current step', [], 'en') }}" namespace="getting_started" />
+
+                <div class="getting-started__current-task stack pb-4">
+                    @include('dashboard.partials.getting-started-individual')
                 </div>
-                <x-interpretation name="{{ __('Next steps', [], 'en') }}" namespace="getting_started" />
-                <ol class="getting-started__task-list stack" role="list">
-                    @stack('next-steps')
-                </ol>
-            </div>
 
-            @if (Auth::user()->checkStatus('approved'))
-                <x-expander :summary="__('Completed steps')" level="3">
-                    <x-interpretation name="{{ __('Completed steps', [], 'en') }}" namespace="getting_started" />
+                <div class="stack pb-4">
+                    <div class="flex items-center gap-5 pt-4">
+                        @svg('heroicon-o-arrow-right', 'icon--xl')
+                        <h3 class="mt-0">{{ __('Next steps') }}</h3>
+                    </div>
+                    <x-interpretation name="{{ __('Next steps', [], 'en') }}" namespace="getting_started" />
                     <ol class="getting-started__task-list stack" role="list">
-                        @stack('completed-steps')
+                        @stack('next-steps')
                     </ol>
-                </x-expander>
+                </div>
+
+                @if ($user->checkStatus('approved'))
+                    <x-expander :summary="__('Completed steps')" level="3">
+                        <x-interpretation name="{{ __('Completed steps', [], 'en') }}" namespace="getting_started" />
+                        <ol class="getting-started__task-list stack" role="list">
+                            @stack('completed-steps')
+                        </ol>
+                    </x-expander>
+                @endif
+            @else
+                <p class="h3">{{ __('Congratulations, your role has been fully approved.') }}</p>
+                @if ($user->individual->isParticipant() && !($user->individual->isConsultant() || $user->individual->isConnector()))
+                    <livewire:prompt :model="Auth::user()" modelPath="dismissed_browse_engagements_prompt_at"
+                        :heading="__('Find engagements to join')" :interpretationName="__('Find engagements to join', [], 'en')" interpretationNameSpace="getting_started" :description="__(
+                            'As a participant, you can answer surveys, or join focus groups, workshops, and more.',
+                        )"
+                        :actionLabel="__('Browse engagements')" :actionUrl="localized_route('engagements.index')" />
+                @endif
+                @if ($user->individual->isConsultant() || $user->individual->isConnector())
+                    <livewire:prompt :model="Auth::user()" modelPath="dismissed_browse_organizations_prompt_at"
```

Comment:
It looks like `dismissed_browse_organizations_prompt_at` may not be wired up to the User model. I don't see it there and am noticing that it doesn't dismiss the prompt properly.

Reply:
I hadn't set it to `fillable` but that's resolved now.